definitions/virtual_machine.md

```diff
@@ -6,10 +6,11 @@ category: Technology
 ## Virtual Machine
 
 ### What it is
-A virtual machine enables a computer system to host other computer systems as guests on the same core system. The host system separates these secondary systems and limits the CPU usage, RAM, and filesystem size and access. With this separation, processes can be executed using different operating systems on the same physical machine.
+A virtual machine enables a computer system to host other computer systems as guests on the same core system. The host system separates these secondary systems and limits the CPU usage, RAM, and filesystem size and access. It will expose virtualised hardware resources (IRQ controllers, Timers, PCI devices) or will emulate them, depending on its capabilities and configuration.
+Virtual Machines allow for a complete operating system and the attached user space isolation from the physical machine. With this separation, processes can be executed using different operating systems and environments, while sharing the same hardware resources.
 
 ### Problem it Addresses
-Previous methods of infrastructure orchestration relied on Bare Metal Machines setup and configured in on premise environments. This method meant taking entire servers offline during upgrades or maintenance. Also, entire servers were required to lie dormant since scaling for demand was not possible. This lack of scalability leads to increased cost and lack of resource utilization.
+Virtual Machines make operating servers at scale easier. Because the whole environment where the workloads are executed is virtualised it can be easily backed up and restored if needed. It can also be easily migrated to another physical machine if maintenance is needed while keeping the service operational and with virtually zero downtime. Sharing the hardware resources between several workloads allows for their more cost effective utilisation. It also enables allocating worker nodes on-demand, sharing the same infrastructure between different tenants.
```

Comment:
This change converts a problem statement, albeit one that needs work, into a "how it helps" statement. This section should stop at illustrating the problem that virtual machines are a response to.